src/js/iotjs.js

```diff
@@ -138,6 +138,9 @@
   process.exitCode = 0;
   process._exiting = false;
   process.emitExit = function(code) {
+    if (typeof(code) !== 'number') {
```

Comment:
I suggest using `typeof` without the bracket pair. `typeof code`.

Reply:
You are right, that's more JS like.
Thank you !